Source/AllKeysOverview.h

```diff
@@ -116,13 +125,14 @@ class AllKeysOverview  : public juce::Component,
 
 	OwnedArray<OctaveBoard> octaveBoards;
 
+	int			currentOctaveSize = 0;

```

Comment:
Only a minor point: maybe we better name this "currentOctaveBoardSize" (or maybe "currentSectionSize, since "Section" is now the sort of official word for it), so it is not confused with a scale size (as in the isomorphic assign dialog).